### ISSUE TITLE
Error running tests with default configuration

### ISSUE THREAD

**Issue by ComputerCarl** [jaredpalmer/razzle#894]
I have created a new project on Windows using `npx create-razzle-app my-app`

When I run `yarn test`, I  get the following message;
```
Determining test suites to run...
--watch is not supported without git/hg, please use --watchAll
npm ERR! Test failed.  See above for more details.
```

**Comment by seekcx** [comment]
same.

**Comment by max-l** [comment]
same problem on ubuntu 18.04

However I can get it to work with: 

```  node node_modules/razzle/scripts/test.js --watchAll```

but the code doesn't run in a Jest context, sinces variables like "document" are undefined.

**Comment by nimaa77** [comment]
fixed in #1080
